webclient/js/elems/raflow.js

```diff
@@ -974,25 +1016,7 @@ window.loadRAPetsGrid = function () {
                 toolbarColumns: false,
                 footer: true,
             },
-            toolbar: {
-                items: [
-                    { type: 'break' },
-                    { type: 'html',  id: 'havePets',
-                        html: function (item) {
-                            var html =
-                              '<div>'+
-                              '<label style="cursor: pointer;">'+
-                              '<input type="checkbox" onchange="" style="margin-right: 10px;" />'+
-                              'Have Pets?'+
-                              '</label>'+
-                              '</div>';
-                            return html;
-                        }
-                    },
-                ]
-            },
-            multiSelect: false,
-            style: 'border: 1px solid black; display: block;',
+            style: 'border: 0px solid black; display: block;',
```

Comment:
add `multiSelect: false` setting back to the code

Reply:
ok